Add debits (#146)
clearhaus/gateway-api-docs: website/content/gateway/api_reference/resources/debits/debits.md

---
title: "debits"
date: 2023-01-05T13:37:00+01:00
anchor: "debits"
weight: 176
---
#### Debits
To transfer money from a cardholder's bank account in real-time, in one step, you make a new debit resource.
```shell
POST https://gateway.clearhaus.com/debits
```

> Debits are created using the same payment methods and parameters as listed under [Authorizations](#authorizations). Exactly one payment method must be used.

Reviewer: ```suggestion
Debits are created using a payment method ([card](#method-card), [applepay](#method-applepay), [googlepay](#method-googlepay), [mobilepayonline](#method-mobilepayonline), [moto](#method-moto), [token](#method-token), or [vipps](#method-vipps)) as listed under [Authorizations](#authorizations). Exactly one payment method must be used.
```

> For authentication and scheme reference to series see the sections under [Authorizations](#authorizations), respectively. Notice that only 3-D Secure version 2, i.e. `[3dsecure][v2][...]`, is supported for debits, not version 1.

Reviewer: ```suggestion
For authentication, [3-D Secure version 2](#authentication-3dsecurev2) is supported, and also [scheme reference to series](#scheme-reference-to-series) is supported---exactly as it is for [Authorizations](#authorizations). Notice that only 3-D Secure version 1 is not supported.
```
